### PR TITLE
update google analytics for server

### PR DESCRIPTION
**Pull request by double-y**
### Description
update google analytics for server

### Checklist

Make sure you have performed following items before submitting this pull request.
If not, please describe the reason.  

- [ ] Add test cases for this fix/enhancement
- [ ] Pass devtools::check()
- [ ] Pass devtools::test()
- [ ] Test installing from github
- [ ] Tested with Exploratory

### CODE REVIEW

File: R/oauth.R [hunk 1]
```diff
@@ -60,11 +60,32 @@ getGoogleTokenForAnalytics <- function(tokenFileId = "", useCache=TRUE){
...
+      validate = "https://www.googleapis.com/oauth2/v1/tokeninfo",
+      revoke = "https://accounts.google.com/o/oauth2/revoke",
+      appname = appName,
+      key = clientId,
+      secret = secret,
+      credentials = list(
+        access_token = token_info$access_token,
+        refresh_token = token_info$token_type,
+        token_type = token_info$token_type,
+        expiry_date = token_info$expiry_data
```
Comment (hideaki) [inline R/oauth.R]: should be $expiry_date instead of $expiry_data

File: R/oauth.R [hunk 2]
```diff
@@ -60,11 +60,32 @@ getGoogleTokenForAnalytics <- function(tokenFileId = "", useCache=TRUE){
...
+      authorize = "https://accounts.google.com/o/oauth2/auth",
+      access = "https://accounts.google.com/o/oauth2/token",
+      validate = "https://www.googleapis.com/oauth2/v1/tokeninfo",
+      revoke = "https://accounts.google.com/o/oauth2/revoke",
+      appname = appName,
+      key = clientId,
+      secret = secret,
+      credentials = list(
+        access_token = token_info$access_token,
+        refresh_token = token_info$token_type,
```
Comment (hideaki) [inline R/oauth.R]: Should be $refresh_token
If this worked, maybe we didn't need refresh_token here in the first place?

File: R/oauth.R [hunk 3]
```diff
@@ -60,11 +60,32 @@ getGoogleTokenForAnalytics <- function(tokenFileId = "", useCache=TRUE){
...
+        token_type = token_info$token_type,
```
Comment (hideaki) [inline R/oauth.R]: Shouldn't this always be "Bearer"? If so we can hard code.

File: R/oauth.R (hunk 2 above)
Comment (double-y) [inline R/oauth.R]: Hmm, it might be true but when I was testing Validation, it was needed. I think having it is safer

File: R/oauth.R (hunk 1 above)
Comment (double-y) [inline R/oauth.R]: Right, will fix it

**Pull request merged**